source/efficient_mode/EfficientWorld.h

```diff
@@ -3,17 +3,30 @@
 
 #include "../default_mode/SymWorld.h"
 #include "../default_mode/DataNodes.h"
+#include "EfficientConfigSetup.h"
 
 class EfficientWorld : public SymWorld {
 private:
+  /**
+    *
+    * Purpose: Represents the configuration settings for a particular run.
+    *
+  */
+  emp::Ptr<SymConfigEfficient> efficient_config = NULL;
+
   /**
     *
     * Purpose: Data node tracking the average efficiency of efficient symbionts.
     *
   */
   emp::Ptr<emp::DataMonitor<double>> data_node_efficiency;
 public:
-  using SymWorld::SymWorld;
+  /**
+   * The constructor for PGGWorld
```

Comment:
Need to fix documentation, also should include full documentation of what the parameters are